cmd/restic/global.go

```diff
@@ -231,6 +231,13 @@ func Verbosef(format string, args ...interface{}) {
 	}
 }
 
+// Verboseff calls Printf to write the message when the verbosity is >= 2
+func Verboseff(format string, args ...interface{}) {
+	if globalOptions.verbosity >= 2 {
```

Comment:
This should be `>= 3` or you’ll add a lot of excess output to `--verbose`. 

Even `Verbose` should be reviewed because it’s quite misleading: `--quiet` is `verbosity = 0`, default is `verbosity = 1`, `--verbose` is `verbosity = 2` and `- -verbose=2` or `-vv` is `verbosity = 3`, but when https://github.com/restic/restic/commit/9fac2ca832459801a730ad3cfcf24af2c23db22d added multiple verbosity levels `Verbose` was converted to `>= 1` so it always prints output unless `--quiet` is specified.

Reply:
I created PR #3009 to fix that.